packages/rolldown/tests/cli/__snapshots__/cli-e2e.test.ts.snap

```diff
@@ -81,6 +81,23 @@ NOTES
   * For more information, please visit https://rolldown.rs/."
 `;
 
+exports[`cli options for bundling > cli default options 1`] = `
+"input cli default options {
+  input: 'index.js',
+  plugins: [
+    {
+      name: 'test-input-plugin',
+      options: [Function: options],
+      outputOptions: [Function: outputOptions]
+    }
+  ]
+}
+output cli default options {}
+<DIR>/index.js  chunk │ size: 0.13 kB
```

Comment:
cc @hyf0 please notice the default cli option snapshot changes if some pr changed it, it shouldn't be changed.

Reply:
Could you give a example PR? I don't get what's going on here.